newm-server/src/main/kotlin/io/newm/server/features/collaboration/model/Collaboration.kt

```diff
@@ -16,7 +16,8 @@ data class Collaboration(
     @Contextual
     val songId: SongId? = null,
     val email: String? = null,
-    var role: String? = null,
+    val role: String? = null, // TODO: STUD-460 - remove "role" field after frontend migrates to use "roles"
```

Comment:
Mark it as `@Deprecated` with a message to use roles instead.

Reply:
Good idea! Done